BIDebugEngine/BIDebugEngine/RVClasses.cpp

```diff
@@ -319,22 +319,28 @@ sourcedocpos tryGetFilenameAndCode(const intercept::types::vm_context::callstack
 
     auto& type = typeid(*&it);
     //#TODO X64 fails on this
-    __debugbreak(); //Some place already has these
+    // __debugbreak(); //Some place already has these
     const auto typeName = ((&type) && ((__std_type_info_data*) &type)->_UndecoratedName) ? type.name() : "TypeFAIL";
     auto hash = type.hash_code();
 
     switch (hash) {
         case CallStackItemSimple::type_hash: {
             auto stackItem = static_cast<const CallStackItemSimple*>(&it);
             //#TODO test if this is the correct instruction or if i should -1 this
```

Comment:
So I guess you tested, and it should be -1?

Reply:
No, I will test to confirm once preprocessor is fixed and line numbers are expected to be correct. I made this change as I hit an overflow here on the last instruction, and using -1 matches the [other](https://github.com/dedmen/ArmaDebugEngine/blob/339c511f5c314f56eafb288a31fa46ecefc4aec2/BIDebugEngine/BIDebugEngine/RVClasses.cpp#L89) [places](https://github.com/dedmen/ArmaDebugEngine/blob/339c511f5c314f56eafb288a31fa46ecefc4aec2/BIDebugEngine/BIDebugEngine/Debugger.cpp#L260) where `sdp` is extracted from `CallStackItemSimple`.  
Is there some reason this particular case is expected to be differnt?